### ISSUE TITLE
"Parkinson's" AND usp8 versus "Parkinsons" AND usp8

### ISSUE THREAD

**Issue by deniseOme** [opentargets/library#20]
This paper got missed out with the current Text mining pipeline and brought to my attention by the attendees of the Dundee workshop:

http://www.pnas.org/content/113/32/E4688.full

"Parkinson's" AND usp8 returns the aforementioned paper but not the search terms "Parkinsons" AND usp8. The latter terms in Google and EuropePMC return the paper (but PubMed does not).

Will the | Library support (some) synonyms?

http://targetvalidation.org/disease/EFO_0002508

**Comment by emepyc** [comment]
Yes, that's key and something we hope will improve over time. It is a very complex task and it will take time until we are able to catch _most_ of them.

**Comment by deniseOme** [comment]
I'm sure it's something far from trivial to do...Things such apostrophes and American versus British spelling could be a good start...No way one can cover all synonyms ever (perhaps not even Google) :)
